.github/workflows/publish.yaml

```
@@ -0,0 +1,37 @@
+---
+# yamllint disable rule:line-length
+name: "Publish to NPM"
+on: # yamllint disable-line rule:truthy
+  release:
+    types:
+      - "published"
+
+jobs:
+  publish:
+    name: Publish to NPM
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v5
+      - uses: "pnpm/action-setup@v4"
+        with:
+          run_install: false
+      - name: "Install Node.js"
+        uses: "actions/setup-node@v6"
+        with:
+          cache: "pnpm"
+      - name: "Install dependencies"
+        run: "pnpm install"
+      # Set the version in package.json to match the tag
+      - name: Write release version
+        run: |
+          VERSION=${GITHUB_REF_NAME#v}
+          echo Version: $VERSION
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+      # NOTE: the flag is necessary because otherwise `npm version <version>` attempts to
+      # cut a git tag with that version, which fails because the git user isn't configured.
+      - run: "pnpm version ${VERSION} --no-git-tag-version"
```

```suggestion
      - run: "pnpm version ${VERSION} --no-git-tag-version"
         id: "get_version"
```

so that line 36 works, no?